packages/autorest.typescript/src/restLevelClient/generateSchemaTypes.ts

```diff
@@ -1,23 +1,15 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import { CodeModel } from "@autorest/codemodel";
 import { Project } from "ts-morph";
-import { generateSchemaTypes as generateSchemaTypesForRLC } from "@azure-tools/rlc-codegen";
-import { transform } from "./transforms/transform";
+import {
+  generateSchemaTypes as generateSchemaTypesForRLC,
+  RLCModel
+} from "@azure-tools/rlc-codegen";
 
 /**
  * Generates types to represent schema definitions in the swagger
  */
-export function generateSchemaTypes(model: CodeModel, project: Project) {
-  // Get all paths
-  const importedParameters = new Set<string>();
-  const importedResponses = new Set<string>();
-  const clientImports = new Set<string>();
-  const rlcModels = transform(model, {
-    importedParameters,
-    importedResponses,
-    clientImports
-  });
+export function generateSchemaTypes(rlcModels: RLCModel, project: Project) {
```

Comment:
why do we still need this function instead of importing it from rlc-codegen directly ?

Reply:
I plan to refactor this to remove project parameter in rlc-codegen after you finish the upgrading. 

Or I could directly refactor this both rlc-codegen & autorest.typescript.

Reply:
ok